nipype/utils/tests/test_nipype2boutiques.py

```diff
@@ -2,16 +2,40 @@
 # emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
 # vi: set ft=python sts=4 ts=4 sw=4 et:
 from future import standard_library
-standard_library.install_aliases()
-
 from ..nipype2boutiques import generate_boutiques_descriptor
+import json
+standard_library.install_aliases()
 
 
 def test_generate():
-    generate_boutiques_descriptor(module='nipype.interfaces.ants.registration',
-                                  interface_name='ANTS',
-                                  ignored_template_inputs=(),
-                                  docker_image=None,
-                                  docker_index=None,
-                                  verbose=False,
-                                  ignore_template_numbers=False)
+    ignored_inputs = [
+        "args",
+        "environ",
+        "output_type"
+    ]
+    desc = generate_boutiques_descriptor(module='nipype.interfaces.fsl',
+                                         interface_name='FLIRT',
+                                         container_image=('mcin/'
+                                                          'docker-fsl:latest'),
+                                         container_index='index.docker.io',
+                                         container_type='docker',
+                                         verbose=False,
+                                         save=False,
+                                         ignore_inputs=ignored_inputs,
+                                         author=("Oxford Centre for Functional"
+                                                 " MRI of the Brain (FMRIB)"))
+
+    with open('utils/nipype2boutiques_example.json', 'r',
```

Comment:
Hi @mgxd, I want to use this example output file in my test case but am not sure how to reference it (what I have above doesn't work). What should the path be?